backend/de.metas.payment.sepa/base/src/main/java/de/metas/payment/sepa/sepamarshaller/impl/SEPAVendorCreditTransferMarshaler_Pain_001_001_03_CH_02.java

```diff
@@ -1003,12 +1015,59 @@ private String createInfo(@NonNull final I_SEPA_Export_Line line)
 		}
 		if (line.getC_BP_BankAccount_ID() > 0)
 		{
-			sb.append(" @C_BP_BankAccount_ID@ ");
-			sb.append(line.getC_BP_BankAccount().getDescription());
+			if (line.getC_BP_BankAccount().getDescription() != null)
```

Comment:
Please, use de.metas.common.util.Check#isNotBlank instead of ``!=null`` when dealing with Strings.